src/engine/renderer/tr_local.h

```diff
@@ -1066,6 +1066,8 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 		stageType_t     type;
 
 		collapseType_t collapseType;
+		// TODO: is it used?
+		int            collapseTextureEnv; // 0, GL_MODULATE, GL_ADD
```

Comment:
Why would you add an unused field?

Reply:
I just moved it from shader struct to stage struct with other collapse stuff. Only the comment “is it used?” was new at this point.

I then deleted that variable with a commit deleting leftovers when I was sure it was not used.

Reply:
oh, right, this is the commit that was supposed to delete it, something bad probably happened on some rebase step.

Reply:
fixed.